code/datums/martial/krav_maga.dm

```diff
@@ -77,12 +18,27 @@
 			neck_chop(A,D)
 			return 1
 		if("leg_sweep")
-			streak = ""
 			leg_sweep(A,D)
 			return 1
-		if("quick_choke")//is actually lung punch
+		if("lung_punch")
+			streak = ""
+			lung_punch(A,D)
+			return 1
+		if("eye_strike")
+			streak = ""
+			eye_strike(A,D)
+			return 1
+		if("unarm")
```

Comment:
Disarm.

Reply:
this is separate, it pulls the item directly out of their hand